src/storage/mvcc/reader/backward_scanner.rs

```diff
@@ -310,18 +315,20 @@ impl<S: Snapshot> BackwardScanner<S> {
 
         // After several `prev()`, we still not get the latest version for the specified ts,
         // use seek to locate the latest version.
-        let last_handled_key = self.write_cursor.key(&mut self.statistics.write).to_vec();
+        assert!(ts > last_checked_commit_ts);
```

Comment:
It's better to put this `assert!` below the previous `if` because its comment said `we must have current commit_ts <= ts`.